css-canvas-medical/styles.css

```diff
@@ -0,0 +1,241 @@
+.container {
+  max-width: 80%;
+  margin: 10px auto;
+  font-family: "normal", Avenir, sans-serif;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  flex-basis: 100%;
+  display: flex;
+}
+
+.column-half {
+  flex-basis: 50%;
+}
+
+#header-link {
+  justify-content: space-evenly;
+}
+
+.header-img img {
+  width: 60%;
+  padding-left: 10%;
+}
+
+.header {
+  padding: 2rem 2rem 1rem;
+  border-bottom: 3px solid #b0b0b0;
+  display: flex;
+  justify-content: space-between;
+}
+
+#button-header {
+  padding: 11px 13px;
+  color: white;
+  background-color: #2a4464;
+  border-radius: 5px;
+}
+
+.column-half a {
+  color: #b0b0b0;
+}
+
+a:link,
+a:visited {
+  text-decoration: none;
+}
+
+.bck-relative {
+  padding: 5%;
+}
+
+.background {
+  position: relative;
+  background-image: url("images/hero.jpg");
+  background-repeat: no-repeat;
+  width: 100%;
+  height: 53vh;
+  background-size: 100%;
+}
+
+.background h2 {
+  font-family: "Tiempos Headline", normal, sans-serif;
+  position: absolute;
+  top: 5vh;
+  left: 9vh;
+  color: white;
+  font-weight: bolder;
+  font-size: 3.3vh;
+}
+
+.background p {
+  position: absolute;
+  top: 18vh;
+  left: 9vh;
+  color: #f3f3f3;
+  font-size: 1.4vh;
+}
+
+.background button {
+  position: absolute;
+  bottom: 10vh;
+  left: 9vh;
+  padding: 1.2vh 3.2vh;
+  background-color: #2a4464;
+  border-radius: 4px;
+  color: white;
+  letter-spacing: 0.2vh;
+  border-color: transparent;
+  font-size: 1.6vh;
+}
```

Comment:
Please change all uses of `vh` to `rem`. This will act very strangely on a viewport with a small height